### PR TITLE
dart-platform and flutter-platform migrate on api

### LINKED ISSUE #516: dart-platform and flutter-platform broken
After redesign https://pub.dev/ this type of badges are broken

dart-platform [Pub points](https://pub.dev/packages/rxdart) /pub/dart-platform/rxdart

dart-platform [Pub points](https://pub.dev/packages/google_sign_in
) /pub/dart-platform/google_sign_in

flutter-platform [Pub points](https://pub.dev/packages/xml) /pub/flutter-platform/xml

### PR DESCRIPTION
**Pull request by dkrutskikh**
close #516

### CODE REVIEW

**Comment by amio** [comment]
Thanks! @dkrutskikh 

We need to rely on apis as much as possible over parsing web.

**Comment by vladimyr** [comment]
@dkrutskikh Please apply these changes: https://github.com/vladimyr/badgen.net/commit/360cab4

Patch download: https://github.com/vladimyr/badgen.net/commit/360cab4.patch

I took your original approach, slightly shuffled things around to make it more readable, and resolved lgtm warning. :tada:

**Comment by vladimyr** [comment]
Also as @amio said, it would be awesome if we could migrate the license route too. Do you happen to know is it possible to fetch it over API?

File: api/pub.ts [hunk 1]
```diff
@@ -5,6 +5,8 @@ import { createBadgenHandler, BadgenError, PathArgs } from '../libs/create-badge
 const PUB_API_URL = 'https://pub.dev/api/'
 const PUB_REPO_URL = 'https://pub.dev/'
 
+const uniq = <T = any>(arr: T[]) => [...new Set(arr)]
```
Comment (vladimyr) [inline api/pub.ts:8]: Sorry, forgot to remove this. :facepalm: 
```suggestion
```

**Comment by dkrutskikh** [comment]
@vladimyr sorry but I don't know how to get info about dart package license.

**Comment by dkrutskikh** [comment]
@vladimyr how I can fix `Vercel` build?

**Comment by vladimyr** [comment]
> @vladimyr how I can fix `Vercel` build?

It's not broken, it is rather unautomated. It'll just stay red until @amio manually deploys a new version. Don't worry about it 😉

**Comment by amio** [comment]
Sorry for the delay, thank you guys @dkrutskikh @vladimyr 
Let's ship it!

**Pull request merged**